config/routes.rb

```diff
@@ -13,6 +13,12 @@
   get 'user/info' => 'users#info'
   get 'user/authenticate' => 'users#authenticate'
 
+  get 'document/v2/open' => 'documents_v2#open'
```

Comment:
Since you are making these custom paths. My preference would be: `v2/document` instead.